pyodide-build/pyodide_build/tests/test_cli.py

```diff
@@ -44,3 +50,59 @@ def test_skeleton_pypi(tmp_path):
     )
     assert result.exit_code != 0
     assert "already exists" in str(result.exception)
+
+
+def test_build_recipe(tmp_path, monkeypatch, request):
+    if request.config.getoption("runtime") == "host":
+        pytest.skip(reason="Can't build recipe without building pyodide first")
```

Comment:
The reason this test fails is that there is no `_sysconfigdata__emscripten_wasm32-emscripten.py` (See https://github.com/pyodide/pyodide/actions/runs/3373970375/jobs/5599098432).

I think we can mock it during the test. But I think we can do that in a separate PR. So no action is required for this PR.